addons/medical_feedback/initSettings.sqf

```diff
@@ -17,4 +17,25 @@
         TRACE_1("painEffectType setting - resetting effect",_this);
         [true] call FUNC(initEffects);
     }
-] call CBA_Settings_fnc_init;
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(bloodVolumeEffectType),
+    "LIST",
+    [LSTRING(BloodVolumeEffectType_DisplayName), LSTRING(BloodVolumeEffectType_Description)],
+    [ELSTRING(medical,Category), LSTRING(SubCategory)],
+    [
+        [FX_BLOODVOLUME_COLOR_CORRECTION, FX_BLOODVOLUME_ICON, FX_BLOODVOLUME_BOTH],
+        [LSTRING(BloodVolumeEffectType_colorCorrection), LSTRING(BloodVolumeEffectType_icon), LSTRING(BloodVolumeEffectType_both)],
+        0
+    ],
+    false,
+    {
+        if (isNil QGVAR(ppPain)) exitWith {
+            TRACE_1("bloodVolumeEffect setting - before postInit",_this);
+        };
+
+        TRACE_1("bloodVolumeEffect setting - resetting effect",_this);
+        [false] call FUNC(initEffects);
```

Comment:
This refreshes all effects. Could be improved to init only the blood ones.

Reply:
Made minor fix. Depending on param initEffects init/reset all effects or just pain & blood volume effects.